app/src/main/java/org/wikipedia/main/MainFragment.java

```diff
@@ -137,6 +138,10 @@ public static MainFragment newInstance() {
             if (getCurrentFragment() instanceof FeedFragment && item.getOrder() == 0) {
                 ((FeedFragment) getCurrentFragment()).scrollToTop();
             }
+            if (getCurrentFragment() instanceof HistoryFragment && item.getOrder() == NavTab.SEARCH.code()) {
+                openSearchActivity(NAV_MENU, null);
+                return true;
```

Comment:
Maybe I was wrong because I am still seeing the content scroll to the top after back to the History page.

Reply:
@cooltey do you mean when you return back from search? during tab change it is not scrolling up.